lib/module_importer.js

```diff
@@ -56,6 +65,7 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
     var relativePath = fragments.slice(1).join("/");
     var pkgRootDir = isRealFile ? importUtils.packageRootDir(path.dirname(prev)) : root;
     var jsFile = importUtils.getModuleByName(moduleName, pkgRootDir);
+    var includePaths = options.includePaths.split(path.delimiter); // surprised this is not already an array, coming from broccoli-eyeglass
```

Comment:
@eoneill I considered that. However, if you look at the test I adde to the suite, I’m passing it in as an array (as expected) and it still totally works. 

There is some magic happening somewhere.

I’m loathe to add extra code paths that may never be needed just because I don’t understand where the magic is happening.

I also think that… if something changes in the future, this line should totally break — now that it's covered by a test, nothing should be hurt by that. And then we can continue on with one code path here.

Sound good?

I can totally remove or update that comment if you want.